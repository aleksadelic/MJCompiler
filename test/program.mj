program MJProgram
	int a[];
	int b[][], c[];
	const int x = 1;
	const int y = 2, z = 3;
	
	class A {
		char c;
		boolean flag;
		int arr[];	
	}
	
	class B {
		int a;
		
		{
		void foo1(int par1, int par2[]) {}
		
		int foo2() {}
		}
	}
	
{
	void main() {
		while (true == 1) {
			continue;
		}
	}
	
	int foo(int a, int b[], char c[][], boolean isGood)
	int locVar, locVar2;
	char locVar3;
	{
		return 1 + 1;
	}
}