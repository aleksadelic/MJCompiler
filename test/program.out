    1    3    5    7    9
-- exception at address 20: expression stack overflow

Completion took 3 ms